src/recommender_system/app/recommender_system/managers/data_manager.py

```diff
@@ -1,2 +1,44 @@
+from typing import Any, Dict, Type
+
+from recommender_system.models.api.attribute import Attribute
+from recommender_system.models.api.attribute_type import AttributeType
+from recommender_system.models.api.base import ApiBaseModel
+from recommender_system.models.api.product import Product
+from recommender_system.models.api.product_translation import ProductTranslation
+from recommender_system.models.api.product_type import ProductType
+from recommender_system.models.api.product_variant import ProductVariant
+
+
 class DataManager:
-    pass
+    def _model_class_from_name(self, model_class_name: str) -> Type[ApiBaseModel]:
+        model_class: Type[ApiBaseModel]
+        if model_class_name == "Attribute":
```

Comment:
Instead of this huge `if`, we may store the models in a dictionary (key: model class names, value: model classes).